content/articles/getting-started-clickfunnels.markdown

```diff
@@ -0,0 +1,222 @@
+---
+title: Getting Started With Your DNSimple Account After Transferring From ClickFunnels
+excerpt: A collection of articles to help you get started with your new DNSimple account quickly.
+categories:
+- DNSimple
+---
+
+# Getting Started With Your DNSimple Account After Transferring From ClickFunnels
```

Comment:
Per our talk - change this as the title since we're covering pre transfer things as well